xcm/src/v0/traits.rs

```diff
@@ -134,6 +134,11 @@ pub trait ExecuteXcm<Call> {
 	/// a basic hard-limit and the implementation may place further restrictions or requirements on weight and
 	/// other aspects.
 	fn execute_xcm(origin: MultiLocation, message: Xcm<Call>, weight_limit: Weight) -> Outcome {
+		log::debug!(
+			target: "xcm::execute_xcm",
+			"origin: {:?}, message: {:?}, weight_limit: {:?}",
+			&origin, &message, weight_limit,
+		);
```

Comment:
log I feel unsure about
Feels useful to be able to just add `-lxcm=debug` and get some measure of logging, but also adds this to (most) every implementation automatically. Also doesn't capture direct calls to `execute_xcm_in_credit` (which e.g. `teleport_assets` does).